nemo_skills/training/data_preparation_utils/filters.py

```diff
@@ -250,20 +288,23 @@ def process_dataset_entry(self, data_entry: str) -> str:
 
 
 class CodeTextFilter(BaseParallelProcessor):
-    def __init__(self, filter_type, solution_key='generation', **kwargs):
+    def __init__(self, filter_type, prompt_config, prompt_template, solution_key='generation', **kwargs):
```

Comment:
I guess it's better to only accept the template, since config is not needed here. And you can use the `load_config` from prompt/utils.py directly to load it